pkg/apis/jenkins/v1alpha2/jenkins_types.go

```diff
@@ -528,6 +528,27 @@ type SeedJob struct {
 	// JenkinsCredentialType is the https://jenkinsci.github.io/kubernetes-credentials-provider-plugin/ credential type
 	// +optional
 	JenkinsCredentialType JenkinsCredentialType `json:"credentialType,omitempty"`
+
+	// GitHubPushTrigger is used for GitHub web hooks
```

Comment:
Fix all comments which tells what that parameters do and add links Jenkins documentation.
Add `// optional` if parameter is optional.

Reply:
`Fix all comments which tells what that parameters do and add links Jenkins documentation.` ???